src/version.h

```diff
@@ -9,9 +9,7 @@
 /**
  * network protocol versioning
  */
-
-
-static const int PROTOCOL_VERSION = 70229;
+static const int PROTOCOL_VERSION = 70230;
```

Comment:
please also bump 
```
static const int MIN_MASTERNODE_PROTO_VERSION = 70227;
```

Reply:
and pls bring back empty lines you deleted here